app/views/admin/articles/_individual_article.html.erb

```diff
@@ -1,55 +1,65 @@
+        <% end %>
+        <% if article.approved %>
+          <span class="c-indicator c-indicator--info">Approved</span>
+        <% end %>
+        <% if article.video %>
+          <span class="c-indicator">Contains video</span>
+        <% end %>
+        <% cache "admin-user-info-#{article.user_id}-#{article.user&.updated_at}", expires_in: 4.hours do %>
+          <% if article.user&.warned? %>
+            <span class="c-indicator c-indicator--danger">User warned</span>
+          <% end %>
+        <% end %>
+      </div>
+    <% end %>
+
+    <% if article.main_image.present? %>
+      <a href="<%= article.path %>" target="_blank" rel="noopener" class="c-link c-link--branded mb-4" style="max-height:150px; max-width: 300px;">
+        <img src="<%= cloud_cover_url(article.main_image) %>" style="background: <%= article.main_image_background_hex_color %>;" alt="cover image" loading="lazy" class="block radius-default h-auto w-100" />
+      </a>
+    <% end %>
+
+    <h2 class="crayons-subtitle-1">
+      <a href="<%= article.path %>" target="_blank" rel="noopener" class="c-link c-link--branded">
         <%= article.title %>
       </a>
     </h2>
 
-    <div>
-      <a href="<%= article.path %>/edit" target="_blank" rel="noopener"><span class="btn btn-sm btn-primary">Edit</span></a>
-
-      <a class="btn btn-sm btn-secondary" href="<%= admin_user_path(article.user_id) %>" target="_blank" rel="noopener">Manage User</a>
-
-      <% if decorated_article.pinned? %>
-        <%= link_to(
-              "Unpin Post",
-              unpin_admin_article_path(decorated_article.id),
-              method: :delete,
-              remote: true,
-              class: "btn btn-sm btn-secondary",
-              data: { "article-target": "unpinButton" },
-            ) %>
-      <% else %>
-          <form method="post" action="<%= pin_admin_article_path(decorated_article.id) %>" class="inline"
-            data-action="submit->article#pinArticle">
-            <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
-            <button type="submit" class="btn btn-sm btn-secondary">
-              Pin Post
-            </button>
-          </form>
-      <% end %>
-
-      <a href="<%= admin_user_path(article.user_id) %>" class="badge badge-light">@<%= article.user&.username %></a>
+    <div class="-ml-1 fs-s flex items-center gap-4">
+      <div class="flex gap-px">
+        <% decorated_article.cached_tag_list_array.each do |tag| %>
+          <a class="crayons-tag crayons-tag--monochrome" href='<%= tag_path(tag) %>'><span class="crayons-tag__prefix">#</span><%= tag %></a>
+        <% end %>
+      </div>
 
-      <span class="badge badge-light">❤️ <%= article.public_reactions_count %> 💬 <%= article.comments_count %></span>
+      <span>❤️ <%= article.public_reactions_count %> likes</span>
+      <span>💬 <%= article.comments_count %> comments</span>
```

Comment:
(not blocking) It's not a big thing, but it looks a bit like there needs to be more space between the emoji and the text:
![emoji and text appear right next to each other despite spacing](https://user-images.githubusercontent.com/20773163/153874818-4a758d97-0f31-4319-bade-d7c38ba47876.png)

Reply:
Weird, it does have spacing on my local env.. Anyway, I made that `&nbsp;` just in case. Dirty but since it's temporary - I don't care :D

Reply:
It still looks the same for me 😕  (using Chrome):
![spacing still small between icon and text](https://user-images.githubusercontent.com/20773163/154078836-7332bafb-1650-4585-a893-ce799822965e.png)

I don't think it's a huge deal though 🤷‍♀️